<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Equipements</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1151</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ajouter machine</string>
     </attribute>
     <widget class="QLabel" name="label_ref">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
color: rgb(255, 255, 255);
background-color: rgb(76, 99, 19);
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
color: rgb(255, 255, 255);
background-color: rgb(76, 99, 19);
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;
</string>
      </property>
      <property name="text">
       <string>disponibilite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
color: rgb(255, 255, 255);
background-color: rgb(76, 99, 19);
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;
</string>
      </property>
      <property name="text">
       <string>date d'achat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
color: rgb(255, 255, 255);
background-color: rgb(76, 99, 19);
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;
</string>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
color: rgb(255, 255, 255);
background-color: rgb(76, 99, 19);
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;
</string>
      </property>
      <property name="text">
       <string>etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
color: rgb(255, 255, 255);
background-color: rgb(76, 99, 19);
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string>Nbr Amortissement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/ajout.png</normaloff>:/mariem/ajout.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_r">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_na">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>400</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/modif.png</normaloff>:/mariem/modif.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>400</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/supp.png</normaloff>:/mariem/supp.png</iconset>
      </property>
     </widget>
     <widget class="QDateEdit" name="le_d">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>4</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>dd-MM-yyyy</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="le_da">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="time">
       <time>
        <hour>12</hour>
        <minute>4</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>camion</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>machine_tri</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>machine_broyage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>machine_stock</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_chercher">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>400</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/recherche.png</normaloff>:/mariem/recherche.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-40</y>
        <width>1261</width>
        <height>721</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="mariem.qrc">:/mariem/ajouuuutttt.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/pdf.png</normaloff>:/mariem/pdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="statistiquesButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/stat.png</normaloff>:/mariem/stat.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_actualiser">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/refresh.png</normaloff>:/mariem/refresh.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tri">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/tri.png</normaloff>:/mariem/tri.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>30</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(76, 99, 19);
font: italic 16pt &quot;Verdana&quot;;
text-decoration: underline;
font: 75 22pt &quot;Bodoni MT&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Formulaire</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(76, 99, 19);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_equipement">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>741</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Statistique</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>521</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>calendrier</string>
       </attribute>
       <widget class="QCalendarWidget" name="calendrierWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>361</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 228, 171);</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditRaison">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>150</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 193, 140);
background-color: rgb(225, 225, 225);</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Marche</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Attente</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Panne</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_back">
      <property name="geometry">
       <rect>
        <x>1012</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mariem.qrc">
        <normaloff>:/mariem/bback-removebg-preview.png</normaloff>:/mariem/bback-removebg-preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>label_ref</zorder>
     <zorder>label_3</zorder>
     <zorder>label_6</zorder>
     <zorder>label_5</zorder>
     <zorder>label_8</zorder>
     <zorder>label_7</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>le_r</zorder>
     <zorder>le_na</zorder>
     <zorder>pb_modifier</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>le_d</zorder>
     <zorder>le_da</zorder>
     <zorder>comboBox1</zorder>
     <zorder>pb_chercher</zorder>
     <zorder>PDF</zorder>
     <zorder>statistiquesButton</zorder>
     <zorder>pb_actualiser</zorder>
     <zorder>pb_tri</zorder>
     <zorder>label_4</zorder>
     <zorder>tabWidget_2</zorder>
     <zorder>comboBox</zorder>
     <zorder>pb_back</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mariem.qrc"/>
 </resources>
 <connections/>
</ui>
